lib/nine_one_one/pager_duty_service.rb

```diff
@@ -1,21 +1,23 @@
 module NineOneOne
   class PagerDutyService
     BASE_HOST = 'events.pagerduty.com'.freeze
-    EVENT_ENDPOINT = '/generic/2010-04-15/create_event.json'.freeze
-
+    EVENTS_API_V2_ENDPOINT = '/v2/enqueue'.freeze
     THROTTLE_HTTP_STATUS = 403
     THROTTLE_RETRIES = 2
+    HIGH_URGENCY_ERROR = 'error'.freeze
+    LOW_URGENCY_ERROR = 'warning'.freeze
 
     def initialize(api_integration_key)
       @api_integration_key = api_integration_key
       @http = Http.new(BASE_HOST)
     end
 
-    def trigger_event(incident_key, description, details_hash = nil)
+    def trigger_event(description, source, details_hash, severity)
```

Comment:
My comment https://github.com/u2i/nine_one_one/pull/5/files#diff-ed21748804f43a3a7b221ff3f11cb5aaR34 applies here as well. I think that we can keep it simple and force only necessary params - in that case `description`.